### PR TITLE
Fix Resurrect Crash

### LINKED ISSUE #5717: Resurrect Crash in MP
### Operating System

Windows x64

### DevilutionX version

Other (please specify version number)

### Describe

commit 81c7d061

Game crashes when Resurrecting players in Multiplayer. Tested in D1.

Call Stack:
image

### To Reproduce

1. Create multiplayer Diablo game
2. Have 2 players in game
3. Cast Resurrect on a dead player

### Expected Behavior

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kphoenix137**
Fixes #5717

### CODE REVIEW

File: Source/spells.cpp [hunk 1]
```diff
@@ -293,6 +293,8 @@ void DoResurrect(size_t pnum, Player &target)
 	target._pMana = 0;
 	target._pManaBase = target._pMana + (target._pMaxManaBase - target._pMaxMana);
 
+	target._pmode = PM_STAND;
```
Comment (glebm) [inline Source/spells.cpp:296]: Can also remove the else branch on 302-304 now

**Review by AJenbo — APPROVED** [review]

**Comment by AJenbo** [comment]
I kinda feel like this should be fixed in `StartStand()` so that this issue can't happen when calling it.

Maybe just swapping these lines?
https://github.com/diasurgical/devilutionX/blob/c99f7cf644907256cdde103b6cda03fe07d5a1d2/Source/player.cpp#L2865

**Comment by glebm** [comment]
The issue is not happening in `StartStand`, it's happening in `CalcPlrInv` (see stack trace and discussion in the linked issue)

**Comment by AJenbo** [comment]
Ok, maybe it should be called after `StartStand()` :D

**Comment by glebm** [comment]
Well, StartStand may not know which graphics to load before the inventory is checked, it's a bit convoluted

**Comment by AJenbo** [comment]
fair point

**Pull request merged**